### PR TITLE
Add classifier attribute to CustomerProvidedDocument

### PR DESCRIPTION
**Pull request by deibys-nuvalence**
## Description
JIRA: https://dol-ewf.atlassian.net/browse/DSG-183

Add classifier to customerProvidedDocument

## How Has This Been Tested?

Please see comment in ticket https://dol-ewf.atlassian.net/browse/DSG-183?focusedCommentId=22710
## Types of changes

Patch
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
Minor
- [X] New feature (non-breaking change which adds functionality)
Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have performed a self-review of my own code
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [ ] I have confirmed all acceptance criteria defined by the story have been met

### CODE REVIEW

**Comment by dtsong** [comment]
Hi @deibys-nuvalence, for this specific PR, if you got a reviewer that grants approval from an app dev perspective, this PR can bypass the `app-work-manager-ephemeral-pr-trigger (dsgov-services)` check.

This is because I accidentally enabled the triggers before having the YAML files ready to go which leads to this situation where it's a false negative. Apologies for the disruption. Let me know via Slack if you have any questions.

File: service/src/main/java/io/nuvalence/workmanager/service/service/TransactionService.java [hunk 1]
```diff
@@ -183,29 +195,85 @@ private Set<UUID> getAllDocuments(final Transaction partialUpdate) {
...
-                                        .active(true)
-                                        .build())
+                        dp -> {
+                            Optional<DocumentClassifierConfiguration> classifier =
+                                    getClassifier(dp, mapDocumentIdToAttributeConfiguration);
+                            return CustomerProvidedDocument.builder()
+                                    .id(dp)
+                                    .reviewStatus(ReviewStatus.NEW)
+                                    .transactionId(transaction.getId())
+                                    .dataPath(transaction.getTransactionDefinition().getSchemaKey())
```
Comment (jgnuvalence) [inline service/src/main/java/io/nuvalence/workmanager/service/service/TransactionService.java:220]: is this correct?, I think this should be the path of the document field, not the schemaKey

**Review by jgnuvalence — CHANGES_REQUESTED** [review]
this looks good to me, I only left 1 comment

File: service/src/main/java/io/nuvalence/workmanager/service/service/TransactionService.java (hunk 1 above)
Comment (jcsuarezm) [reply service/src/main/java/io/nuvalence/workmanager/service/service/TransactionService.java:220]: Hi @deibys-nuvalence and @jgnuvalence, I am currently working on this for my bug fix ticket.

Comment (jgnuvalence) [reply service/src/main/java/io/nuvalence/workmanager/service/service/TransactionService.java:220]: ok makes sense, thank you @jcsuarezm

**Comment by jgnuvalence** [comment]
you just need to update the branch with main

**Review by jgnuvalence — APPROVED** [review]

**Pull request merged**